Cargo.toml

```diff
@@ -11,11 +11,16 @@ version = "0.1.0"
 edition = "2021"
 license = "Apache-2.0"
 repository = "https://github.com/astarte-platform/astarte-device-sdk-rust"
+description ="Astarte Device SDK"
```

Comment:
Changes to "A library that provides communication and pairing primitives to an Astarte Cluster".